RandLAPACK/drivers/rl_revd2.hh

```diff
@@ -113,61 +113,59 @@ class REVD2 : public REVD2alg<T> {
 };
 
 // -----------------------------------------------------------------------------
-
+/// Power scheme for error estimation, based on algorithm E.1 from https://arxiv.org/pdf/2110.02820.pdf.
+/// This routine is too specialized to be included into RandLAPACK::utils
+/// p - number of algorithm iterations
+/// vector_buf - buffer for vector operations
+/// Mat_buf - buffer for matrix operations
+/// All other parameters come from REVD2
 template <typename T>
 T power_error_est(
     int64_t m,
     int64_t k,
     int p,
-    T* Omega_dat,
+    T* vector_buf_dat,
     T* V_dat,
     blas::Uplo uplo,
     T* A_dat,
-    T* Y_dat,
+    T* Mat_buf_dat,
```

Comment:
I suggest you remove the "dat" suffixes from the relevant variables here. You have that suffix in the calling function because the calling function obtains these as ``.data()`` of ``std::vector<T>`` variables. Once you factor out a function from its context it's good to rename variables so that the abstract context is here. You already did that by renaming ``Y_dat`` and ``Omega_dat``; let's take it one step further.

Reply:
Fixed.